packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -380,6 +403,7 @@ private void onSignInResult(GoogleSignInResult result) {
       if (result.isSuccess()) {
         GoogleSignInAccount account = result.getSignInAccount();
         currentAccount = account;
+        registrar.publish(account);
```

Comment:
We were planning to deprecate the value publishing API. Can't potential clients use the GoogleSignIn API directly to get the account?